webui/react/TODO.md

```diff
@@ -0,0 +1,60 @@
+# Product Week
```

Comment:
I'll remove this unless we want something like this as opposed to inline todos. (or maybe we rather not leave any TODOs in git)

Reply:
https://determinedai.atlassian.net/browse/DET-5382